### PR TITLE
kde-plasma/plasma-meta: update dependencies

### PR DESCRIPTION
**Pull request by rachid-a**

### CODE REVIEW

File: kde-plasma/plasma-meta/plasma-meta-9999.ebuild [hunk 1]
```diff
@@ -31,7 +31,6 @@ RDEPEND="
 	>=kde-plasma/ksshaskpass-${PV}:${SLOT}
 	>=kde-plasma/ksystemstats-${PV}:${SLOT}
 	>=kde-plasma/kwayland-${PV}:${SLOT}
-	>=kde-plasma/kwayland-integration-5.27.9:5
```
Comment (a17r) [inline kde-plasma/plasma-meta/plasma-meta-9999.ebuild:34]: Why do you drop it?

**Review by a17r — CHANGES_REQUESTED** [review]

File: kde-plasma/plasma-meta/plasma-meta-9999.ebuild (hunk 1 above)
Comment (rachid-a) [reply kde-plasma/plasma-meta/plasma-meta-9999.ebuild:34]: The source is currently only QT 5

https://invent.kde.org/plasma/kwayland-integration

Comment (thesamesam) [reply kde-plasma/plasma-meta/plasma-meta-9999.ebuild:34]: https://invent.kde.org/plasma/kwayland-integration/-/merge_requests/61

Comment (a17r) [reply kde-plasma/plasma-meta/plasma-meta-9999.ebuild:34]: Rebase on top of HEAD, dropping this change, and make this commit only about DESCRIPTION and kde-apps/khelpcenter:6, then we can merge it.

**Review by a17r — CHANGES_REQUESTED** [review]

**Comment by a17r** [comment]
You need to use `git pull --rebase origin master`, we disallow merge commits in ebuild repositories. :)

File: kde-plasma/plasma-meta/plasma-meta-9999.ebuild (hunk 1 above)
Comment (a17r) [reply kde-plasma/plasma-meta/plasma-meta-9999.ebuild]: See also my comment from commit c79492bca3c28e690c114fac53b1feb5151ac58a.

May be made optional in the future but makes no sense right now.

**Comment by rachid-a** [comment]
I will create new request

**Pull request closed without merging**